### PR TITLE
Pick up prestosql version 340

### PR DESCRIPTION
**Pull request by meneal**
Supports IBM/docker-prestodb#33

### CODE REVIEW

**Comment by meneal** [comment]
As seen [here](https://github.com/IBM/presto-db2/pull/52/checks?check_run_id=985252955#step:5:28), Maven is angry about the order of deps.

File: pom.xml [hunk 1]
```diff
@@ -46,8 +46,8 @@
 		</dependency>
 
 		<dependency>
-			<groupId>io.prestosql</groupId>
-			<artifactId>presto-base-jdbc</artifactId>
+				<groupId>javax.validation</groupId>
+				<artifactId>validation-api</artifactId>
```
Comment (shawnzhu) [inline pom.xml:50]: would you please reformat it by removing whitespaces/tabs?

**Comment by shawnzhu** [comment]
The requirement of dependencies order is introduced here: https://github.com/prestosql/presto/pull/4534

File: pom.xml (hunk 1 above)
Comment (meneal) [reply pom.xml]: done

**Pull request merged**